docs/reference/buildx_du.md

```diff
@@ -117,3 +118,41 @@ uj802yxtvkcjysnjb4kgwvn2v         true           11.68MB       45 hours ago
 Reclaimable:    2.627GB
 Total:          2.627GB
 ```
+
+### <a name="format"></a> Format the output in verbose mode (--format)
+
+The formatting options (`--format`) can be used with [`--verbose` flag](#verbose)
+to print the output in `raw` (default) or `json`:
+
+```console
+$ docker buildx du --verbose --format=json
+[
+  ...
+  {
```

Comment:
FWIW, as this is brand new, I'd be tempted to not pretty-print the json and just 1 entry per line; users can still `jq` to pretty-print the JSON